ec2/ec2test/network_interfaces.go

```diff
@@ -404,6 +416,10 @@ func (srv *Server) createIFace(w http.ResponseWriter, req *http.Request, reqId s
 	return resp
 }
 
+func (srv *Server) dnsNameFromPrivateIP(privateIP string) string {
+	return fmt.Sprintf("ip-%s.ec2.internal", strings.Replace(privateIP, ".", "-", -1))
```

Comment:
Using .internal may actually resolve to something when running tests, whereas .invalid is reserved, see rfc 2606. The tests should not be trying to do dns lookups on these faked names, but I like using .invalid or .test even so.